<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureMenu</class>
 <widget class="QScrollArea" name="TemperatureMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScrollArea</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>275</width>
     <height>713</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_Fine">
      <property name="title">
       <string>Temp.</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Icons">
           <item>
            <widget class="QLabel" name="label_Warm">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/img/icons/temperatureMenu/warm.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_IconTitle">
             <property name="text">
              <string>Source Temperature</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_Cool">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/img/icons/temperatureMenu/cold.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_Temperature">
           <property name="minimum">
            <number>1850</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="pageStep">
            <number>270</number>
           </property>
           <property name="value">
            <number>5500</number>
           </property>
           <property name="sliderPosition">
            <number>5500</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_Low">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>1850K</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_High">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>10000K</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_Coarse">
      <property name="title">
       <string>Presets</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="12" column="0">
        <widget class="QRadioButton" name="radioButton_6500K">
         <property name="text">
          <string>6500K</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Computer Monitor</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QRadioButton" name="radioButton_4500K">
         <property name="text">
          <string>4500K</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Tungsten Halogen</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Soft White Incandescent</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QRadioButton" name="radioButton_7200K">
         <property name="text">
          <string>7200K</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Horizon Daylight</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="radioButton_2550K">
         <property name="text">
          <string>2550K</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="radioButton_2400K">
         <property name="text">
          <string>2400K</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QRadioButton" name="radioButton_2700K">
         <property name="text">
          <string>2700K</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QRadioButton" name="radioButton_5000K">
         <property name="text">
          <string>5000K</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QRadioButton" name="radioButton_3200K">
         <property name="text">
          <string>3200K</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Cool White Flourescent</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="radioButton_1850K">
         <property name="text">
          <string>1850K</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Overcast Daylight</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Soft White Flourescent</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Clear Blue Sky</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QRadioButton" name="radioButton_10000K">
         <property name="text">
          <string>10000K</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Candle Flame</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Standard Incandescent</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_19">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
